### PR TITLE
Deprecate appinfo/update.php - RepairSteps within info.xml are to be …

### PR DESCRIPTION
**Pull request by DeepDiver1975**
…used

### CODE REVIEW

File: core/Command/App/CheckCode.php [hunk 1]
```diff
@@ -170,6 +170,9 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$errors = array_merge($errors, $infoErrors);
 		}
 
+		$infoErrors = $this->analyseUpdateFile($appId, $output);
+		$errors = array_merge($errors, $infoErrors);
```
Comment (nickvergessen) [inline core/Command/App/CheckCode.php]: Having a deprecated feature should not make the app fail on travis in my opinion....

So I'm against returning 101 in that case

File: core/Command/App/CheckCode.php [hunk 2]
```diff
@@ -178,4 +181,26 @@ protected function execute(InputInterface $input, OutputInterface $output) {
...
+		if ($appPath === false) {
+			throw new \RuntimeException("No app with given id <$appId> known.");
+		}
+
+		$updatePhp = $appPath . '/appinfo/update.php';
+		if (file_exists($updatePhp)) {
+			$output->writeln("<info>Deprecated file found: $updatePhp - please use repair steps</info>");
+		}
+
+		return [
```
Comment (nickvergessen) [inline core/Command/App/CheckCode.php]: should be inside the if?

**Comment by DeepDiver1975** [comment]
@nickvergessen addressed - THX a lot!

**Comment by nickvergessen** [comment]
```
$ ./occ app:check-code activity
Deprecated field available: shipped => true
Deprecated file found: /home/nickv/ownCloud/master/core/appsrepos/activity/appinfo/update.php - please use repair steps
App is compliant - awesome job!

$ ./occ app:check-code systemtags_management
App is compliant - awesome job!
```

:+1:

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Pull request merged**